src/f_serial.hpp

```diff
@@ -43,8 +43,10 @@ void processSerialData()
             if (buffer[0] == 0x06)
             {
                 LOGV1(DEBUG_SERIAL, F("[SERIAL]: Received: ACK request, replying 1"));
-    #if DEBUG_LEVEL == DEBUG_NONE
-                Serial.print('1');
+                // When not debugging, print the result to the serial port .
+                // When debugging, only print the result to Serial if we're on seperate ports.
+    #if (DEBUG_LEVEL == DEBUG_NONE) || (DEBUG_SEPARATE_SERIAL == 1)
+                Serial.print("1");
```

Comment:
The implementation has changed here I think, I'm not too sure what this code is supposed to be doing but you've changed the serial response from `0x31` (`'1'`) to `0x31 0x0` (`'1'` with null termination)

Reply:
Oops, good catch.